src/Microsoft.PowerShell.ConsoleGuiTools/Microsoft.PowerShell.ConsoleGuiTools.csproj

```diff
@@ -1,12 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <Import Project="..\..\GraphicalTools.Common.props" />
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net7.0</TargetFramework>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <!-- To pull Terminal.Gui from a local nuget source:                        
+        - Build Terminal.Gui locally in ../../gui-cs/Terminal.Gui          
+        - Change Terminal.Gui Version= to "major.minor.patch-*"     
+        - Add ';https://api.nuget.org/v3/index.json' to the end of the RestoreSources property group below
+        - Uncomment the RestoreSources property group below
+        -->
+        <!-- <RestoreSources>$(RestoreSources);../../../gui-cs/Terminal.Gui/Terminal.Gui/bin/Debug</RestoreSources> -->
     </PropertyGroup>
 
     <ItemGroup>
         <PackageReference Include="Nstack.Core" Version="1.0.7" />
-        <PackageReference Include="Terminal.Gui" Version="1.13.5" />
+        <PackageReference Include="Terminal.Gui" Version="1.13.*" />  
```

Comment:
This should be whatever the latest version is, not `.*` because Dependabot is going to come along and replace it.